src/App.vue

```diff
@@ -19,11 +22,15 @@ export default defineComponent({
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap");
 @import "./src/styles/_variables.scss";
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
+  margin: 0 auto;
+  font-family: "Poppins", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 18px;
   text-align: center;
-  background: #ecedf4;
+  color: hsl(257, 27%, 26%);
```

Comment:
There is `_variables.scss` file with colors:
write it: `color: $darkViolet`

Reply:
I am not sure about that we can use it.
I have once problem that IE doesn't support css variables

Reply:
Flexbox also isn't fully supported by IE. It doesn't matter. I think it doesn't matter...